app/script.js

```diff
@@ -0,0 +1,47 @@
+// //Quiz 5
+
+$(function() {
+ $("button#joke").click(function() {
+ $.get("./joke", function(response) {
+ var joke = response.setup + " ... " + response.punchline;
+ $("body").text(joke);
```

Comment:
this replaces your whole page with your response. Might be what you wanted, but it's a little funky from a user perspective. :)